clouddriver-web/clouddriver-web.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
   compile spinnaker.dependency('korkSwagger')
   compile spinnaker.dependency('bootActuator')
   compile spinnaker.dependency('bootDataRest')
-  testCompile ("org.springframework.boot:spring-boot-starter-test")
+  testCompile ("org.springframework.boot:spring-boot-starter-test:${spinnaker.version('springBoot')}")
```

Comment:
Same comment as other PR, `testCompile spinnaker.dependency('bootTest')` should work here